src/org/labkey/test/components/DomainDesignerPage.java

```diff
@@ -67,22 +65,22 @@ public boolean isAlertVisible()
         return Locators.alert.findOptionalElement(getDriver()).map(WebElement::isDisplayed).orElse(false);
     }
 
-    public WebElement saveButton()
+    public WebElement finishButton()
     {
-        return elementCache().saveButton();
+        return elementCache().finishButton();
     }
 
-    public DomainFormPanel fieldProperties()
+    public DomainFormPanel fieldsPanel()
```

Comment:
Maybe a comment to say that this just returns the first one if there are multiple?